File: planeshapes.py
print("--- 4. Area of Plane Shapes ---")
# Rectangle
length_rect = float(input("Enter rectangle length: "))
width_rect = float(input("Enter rectangle width: "))
print(f"Rectangle Area: {length_rect * width_rect:.2f}\n")

# Triangle
base_tri = float(input("Enter triangle base: "))
height_tri = float(input("Enter triangle height: "))
print(f"Triangle Area: {0.5 * base_tri * height_tri:.2f}\n")

# Circle
radius = float(input("Enter circle radius: "))
print(f"Circle Area: {3.14159 * (radius ** 2):.2f}\n")

File: planeshapes.cpp
cout << "--- 4. Area of Plane Shapes ---\n";
    // Rectangle
    double length_rect, width_rect;
    cout << "Enter rectangle length: ";
    cin >> length_rect;
    cout << "Enter rectangle width: ";
    cin >> width_rect;
    cout << "Rectangle Area: " << (length_rect * width_rect) << "\n\n";

    // Triangle
    double base_tri, height_tri;
    cout << "Enter triangle base: ";
    cin >> base_tri;
    cout << "Enter triangle height: ";
    cin >> height_tri;
    cout << "Triangle Area: " << (0.5 * base_tri * height_tri) << "\n\n";

    // Circle
    double radius;
    cout << "Enter circle radius: ";
    cin >> radius;
    cout << "Circle Area: " << (3.14159 * radius * radius) << "\n\n";